cli/parameter.go

```diff
@@ -127,3 +127,20 @@ func parseParameterMapFile(parameterFile string) (map[string]string, error) {
 	}
 	return parameterMap, nil
 }
+
+// buildDebugFlags contains options relating to troubleshooting build issues.
+type buildDebugFlags struct {
+	provisioner bool
+}
+
+func (bdf *buildDebugFlags) cliOptions() []serpent.Option {
+	return []serpent.Option{
+		{
+			Flag: "debug-provisioner",
```

Comment:
We're not so much debugging the provisioner as we're debugging the build via increased provisioner logs. I think we can do a bit better on the naming but I don't have any great ideas. Maybe `--debug-build`?

Reply:
I could just straight-up make it `--provisioner-log-debug`? WDYT?

Reply:
That still sounds like provisioner debugging IMO, not build debugging 😅